Kong/docs.konghq.com
# Chore: AI Gateway docs cleanup

### Description

Making a few adjustments to improve the AI Gateway docs:
* Adding missing Content Safety plugin to AI Gateway overview
* Creating an include for plugin links
* Moving all the AI Proxy LLM provider guides into a subfolder for better nav structure:
Before & after:
<img src="https://github.com/Kong/docs.konghq.com/assets/54370747/887b54fd-6c34-4c9e-9caf-d8f19b2aa8e1" style="width:50%">

<img src="https://github.com/Kong/docs.konghq.com/assets/54370747/cf02dcbe-c61d-4299-b668-dcbfdeba742f" style="width:50%">

### Testing instructions

Preview link: 
https://deploy-preview-7416--kongdocs.netlify.app/hub/kong-inc/ai-proxy/unreleased/how-to/llm-provider-integration-guides/anthropic/ - see sidenav
https://deploy-preview-7416--kongdocs.netlify.app/gateway/unreleased/ai-gateway/#ai-usage-governance

### Checklist
